File: Blackjack Project/Part 1B/value.cpp
#include<iostream>

using namespace std;

int main()
{
    int card;
    cin >> card;
    if(card == 1){
        cout << "Your hand value is 11." << endl;
    } else if(card >= 2 && card <= 10){
        cout << "Your hand value is " << card << "." << endl;
    } else if(card >= 11 && card <= 13){
        cout << "Your hand value is 10." << endl;
    } else{
        cout << "BAD CARD";
    }
    return 0;
}

File: Blackjack Project/Part 1B/value.py
card = int(input())
if card == 1:
    print("Your hand value is 11.")
elif card >= 2 and card <= 10:
    print("Your hand value is " + str(card) + ".")
elif card >= 11 and card <= 13:
    print("Your hand value is 10.")
else:
    print("BAD CARD")
